swift3/schema/lifecycle_configuration.rng

```diff
@@ -85,8 +85,29 @@
             </element>
           </interleave>
         </element>
-      </zeroOrMore>
-    </interleave>
+      </interleave>
```

Comment:
When there is no `And`, only one filter is accepted; either a `Prefix` or a `Tag`.
So I think the first `interleave` (and `optional`) is not useful.
```xml
    <choice>
      <element name="Prefix">
        <data type="string"/>
      </element>
      <element name="Tag">
        <interleave>
          <element name="Key">
            <data type="string"/>
          </element>
          <element name="Value">
            <data type="string"/>
          </element>
        </interleave>
      </element>
      <element name="And">
        [...]
      </element>
    </choice>
```